eclipse-theia/theia-blueprint
# Theia 1.30.0

#### What it does
* updates Theia and dependencies
* adjust test case
* switch to `@theia/git` for now to get a working panel (https://github.com/eclipse-theia/theia-blueprint/issues/232)

#### How to test
Build Blueprint and verify that is uses the new Theia version and that Blueprint is working as expected. 

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
